spec/helpers/datasets_helper_spec.rb

```diff
@@ -22,4 +22,16 @@
       expect(helper.contact_email_is_email?(dataset)).to be_falsey
     end
   end
+
+  describe "#foi_email_is_email?" do
+    it "returns true when the email is valid" do
```

Comment:
Nitpick: I'd be a bit more explicit with the description. You're just checking for an @ so a@b would pass your test but it's not a valid email address. Something like "returns true when the email address contains a @"